### DISCUSSION TITLE
Trying to bind Ikea Tradfri Wireless Dimmer to Hue Bulb

### DISCUSSION THREAD

**Discussion by babecassis** [Koenkk/zigbee2mqtt#11503] [Devices]
Has anyone successfully binded (bound?) a tradfri wireless dimmer (https://www.zigbee2mqtt.io/devices/ICTC-G-1.html) to a Hue bulb (https://www.zigbee2mqtt.io/devices/9290022268.html#philips-9290022268) and gotten dimming to work? I have tried through the UI as well as via mqtt and the binding reports success, but using the dimmer has no action.
<img width="1299" alt="image" src="https://user-images.githubusercontent.com/72462662/154828357-9c5b4030-7940-4630-8f5f-285e7d5b103e.png">

**Comment by miguelarios** [comment]
Did you ever get this going?

**Reply by babecassis** [reply]
Unfortunately, no. I ended up Tradfri bulbs

**Reply by miguelarios** [reply]
So you found the tradfri bulbs bind way better to this remote?

**Reply by babecassis** [reply]
They did! The only issue "complaint" is that the Tradfri bulbs dont dim very low via the remote.

**Comment by peterjochum** [comment]
This now seems to work out of the box. Using IKEA Tradfri remote (2xAAA) to on/off and dim a HUE 1600lm E27 white bulb.

- Bound LevelCtrl and OnOff
- Holding up(on) dims higher
- Holding down(off) dims lower

I am also dimming another group of lights consisting of two HUE 800lm E27 using a Tradfri remote.
